Source/Scripts/cheat_core_player.lua

```diff
@@ -108,147 +87,336 @@ function cheat:teleport_to(line)
+  local args = cheat:argsProcess(line, cheat.cheat_tp_bh_args)
+  local nplace, nplaceErr = cheat:argsGet(args, 'place')
+  
+  local places = {}
+  places["LEDETCHKO"] = "x:2042 y:1282 z:28"
+  places["RATTAY"] = "x:2380 y:595 z:31"
+  places["SASAU"] = "x:959 y:1317 z:19"
+  places["TALMBERG"] = "x:2390 y:2810 z:89"  
+  
+  if gender ~= 2 then
+    if not nplaceErr then
+      if places[cheat:toUpper(nplace)] ~= nil then
+        cheat:teleport(places[cheat:toUpper(nplace)])
+      else
+        cheat:logError("Invalid Place - See list of supported places type: 'cheat_tp_bh ?'")
+      end
+    end
+  else
+    cheat:logError("You can't use this command while playing Thereza!")
+  end
+end
+
+-- ============================================================================
+-- cheat_tp_cb (CHARCOAL-BURNER CAMPS)
+-- ============================================================================
+cheat.cheat_tp_cb_args = {
+  place = function(args,name,showHelp) return cheat:argsGetRequired(args, name, showHelp, "Teleport to charcoal-burner camp") end,
+}
+
+cheat:createCommand("cheat_tp_cb", "cheat:tp_cb(%line)", cheat.cheat_tp_cb_args,
+"Teleports the player to the given place. Supported places (case insensitive):\n$8NH_NORTH, NH_SOUTH, RATTAY, ROVNA, SASAU, TALMBERG",
+"Example", "cheat_tp_cb place:RATTAY")
+function cheat:tp_cb(line)
+  local gender = player.soul:GetGender()
+  local args = cheat:argsProcess(line, cheat.cheat_tp_cb_args)
+  local nplace, nplaceErr = cheat:argsGet(args, 'place')
+  
+  local places = {}
+  places["NH_NORTH"] = "x:3256 y:2122 z:127"
+  places["NH_SOUTH"] = "x:3634 y:942 z:86"
+  places["RATTAY"] = "x:1909 y:677 z:28"
+  places["ROVNA"] = "x:1316 y:3316 z:62"
+  places["SASAU"] = "x:343 y:1981 z:18"
+  places["TALMBERG"] = "x:2695 y:2317 z:102"  
+  
+  if gender ~= 2 then
+    if not nplaceErr then
+      if places[cheat:toUpper(nplace)] ~= nil then
+        cheat:teleport(places[cheat:toUpper(nplace)])
+      else
+        cheat:logError("Invalid Place - See list of supported places type: 'cheat_tp_cb ?'")
+      end
+    end
+  else
+    cheat:logError("You can't use this command while playing Thereza!")
+  end
+end
+
+-- ============================================================================
+-- cheat_tp_gn (FOREST GARDEN)
+-- ============================================================================
+cheat.cheat_tp_gn_args = {
+  place = function(args,name,showHelp) return cheat:argsGetRequired(args, name, showHelp, "Teleport to garden") end,
+}
+
+cheat:createCommand("cheat_tp_gn", "cheat:tp_gn(%line)", cheat.cheat_tp_gn_args,
+"Teleports the player to the given place. Supported places (case insensitive):\n$8GN_1, GN_2, GN_3, GN_4, GN_5, GN_6, GN_7, GN_8, GN_9, GN_10, GN_11, GN_12, GN_13, GN_14, GN_15,\n$8GN_16, GN_17, GN_18, GN_19, GN_20, GN_21, GN_22, GN_23, GN_24, GN_25, GN_26, GN_27, GN_28, GN_29, GN30, GN_31, GN_32, GN_33, GN_34, GN_35, GN_36, GN_37, GN_38, GN_39",
```

Comment:
Suggest same fix here as well.

Reply:
Seems like switching to just numbers would be good here. Simplifies the docs to just a number range.